cypress/e2e/po/components/cluster-badge.po.ts

```diff
@@ -0,0 +1,53 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import CheckboxInputPo from '@/cypress/e2e/po/components/checkbox-input.po';
+import ColorInputPo from '@/cypress/e2e/po/components/color-input.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+
+export default class ClusterBadgePo extends ComponentPo {
```

Comment:
This is the PO for the custom badge dialog, so just needs a rename to make it clear